### ISSUE TITLE
Implement a function to select CAN-HW

### ISSUE THREAD

**Issue by Nahrain1** [amosproj/amos2024ss07-updating-flash-boot-loader#17]
## User story
1. As a user 
2. I want to be able to select the desired CAN hardware
3. So that I can configure the CAN interface for communicating and updating the ECU Application

## Acceptance criteria
* The UI includes a drop-down menu listing available CAN hardware channels 
* The user can select a CAN hardware channel from the drop-down menu
* CAN Speed of 33.3, 50, 83.3, 100, 125, 250, 500, 1000 kBit/s are supported
* CAN-FD Speed of 1000kBit/s up to 8000kBit/s in 1000kBit/s steps are supported
* Optionally: Up to 4 MBPS with CAN-FD

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
